functions/main.py

```diff
@@ -54,6 +54,11 @@ def convert_tos_to_json(tree: nx.DiGraph) -> Dict[str, List[Dict]]:
         )
         output[section] = data
 
+    if "branch" in output:
+        for i in range(1, 4):
```

Comment:
let's use `[1, 2, 3]` instead of `range(1, 4)` here for extra clarity.

Reply:
Of course :+1: